src/Ssh/Ssh/Az.Ssh.psd1

```diff
@@ -0,0 +1,131 @@
+#
+# Module manifest for module 'Az.Ssh'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 8/22/2019
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '0.0.1'
+
+# Supported PSEditions
+CompatiblePSEditions = 'Core', 'Desktop'
+
+# ID used to uniquely identify this module
+GUID = '91832aaa-dc11-4583-8239-bce5fd531604'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Microsoft Azure PowerShell - cmdlets for connecting to Azure VMs using SSH in Windows PowerShell and PowerShell Core.'
+
+# Minimum version of the PowerShell engine required by this module
+PowerShellVersion = '5.1'
+
+# Name of the PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+DotNetFrameworkVersion = '4.7.2'
```

Comment:
We normally target `4.6.1` for WindowsPowerShell 5.1 compatibility.

Reply:
I will confirm with Azure PowerShell if that is the case for their modules as well. I set it to 4.7.2 because other Az PowerShell modules all do the same.